examples/IBM Db2/dialect.yml

```diff
@@ -329,6 +332,7 @@ map to field types:
 column types:
   integer: int
   float: float
+  fixed: decimal(38, 8)
```

Comment:
shouldn't that be "decimal(31,8)" ?

Reply:
good catch.  I just made the correction. Do you think we can merge this PR?